src/Eshopworld.WorkerProcess/CosmosDistributedLock/CosmosDistributedLockStore.cs

```diff
@@ -24,7 +25,7 @@ public CosmosDistributedLockStore(IDocumentClient documentClient, IOptions<Cosmo
             _options = options;
             _telemetry = telemetry;
             _retryPolicy = CreateRetryPolicy();
```

Comment:
are we retrying forever if a failure occurs? Shouldn't we give up after a certain number of times

Reply:
I agree. Do you think a 3 or 5 time retry limit would be good?

Reply:
we could every 200 ms for 5 times?